app-text/scdoc/scdoc-1.5.2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="Standalone tool for generating man pages with a simple syntax"
+HOMEPAGE="https://git.sr.ht/~sircmpwn/scdoc"
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://git.sr.ht/~sircmpwn/scdoc"
+	inherit git-r3
+else
+	SRC_URI="https://git.sr.ht/~sircmpwn/scdoc/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+
+src_prepare() {
+	default
+
+	sed -e 's/-Werror//' \
+		-i Makefile || die 'Failed to patch Makefile'
+}
+
+src_compile() {
+	MY_HS="./scdoc"
```

Comment:
I don't think this variable needs to be global.

Reply:
Fair. I just kept what was there in `scdoc-1.3.1.ebuild` and only tinkered enough to make
it work, not to make it perfect as far as style goes.

Reply:
fixed.